website/docs/docs/develop/develop-in-the-cloud.md

```diff
@@ -0,0 +1,249 @@
+
+To generate your project’s documentation (docs) in the IDE, run `dbt docs generate` in the command bar. This command generates the docs for your dbt project as it currently exists in development.
+
+After you generate a successful run, you can view your documentation for your dbt project in real time by clicking **View Docs** or the **Book** icon on above the file tree. 
+
+You can view the latest version of your documentation rendered in a new browser window, and inspect and verify what your project's documentation will look like before you deploy your changes to production.
+
+
+## Develop in the Cloud IDE (beta)
+
+:::info Join our beta
+
+
+If you’d like to try the dbt Cloud IDE beta, available for multi-tenant instances, please [sign up](https://docs.google.com/forms/d/e/1FAIpQLSdlU65gqTZPyGAUc16SkxqTc50NO9vdq_KGx1Mjm_4FB_97FA/viewform) to join the beta. To learn more about the beta features, you can read this section.
+
+:::
+
+### Prerequisites
+
+To develop in dbt Cloud IDE (beta), you need to meet these requirements:
+
+- Your dbt project must be compatible with dbt v0.15.0. The dbt IDE is powered by the [dbt-rpc](/reference/commands/rpc) which was overhauled in dbt v0.15.0
+- You must have a [Developer License](/docs/dbt-cloud/access-control/cloud-seats-and-users)
+- - Your dbt project must be connected to a [data platform](/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database)
+- You must have a [development environment and development credentials](/docs/develop/develop-in-the-cloud#set-up-the-cloud-ide) set up
+- The environment must be on dbt version 1.0 or higher
+- Currently only multi-tenant instances of dbt Cloud can develop in the updated beta version of the Cloud IDE
+    * Single-tenant instances will soon be able to opt into this Beta release.
+- Your dbt repository (in dbt Cloud) must have write access enabled
+    * See [Connecting your GitHub Account](/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application) and [Importing a project by git URL](/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-import-a-project-by-git-url) for detailed setup instructions
+
+
+The dbt Cloud IDE beta brings the startup and interaction time for dbt project development down from minutes to seconds, includes reliable performance and delightful enhancements to help you develop more efficiently!
+
+### Start up and work retention
+
+<table>
+<tr><th>Start up process </th><th>Work retention</th></tr>
+<tr><td>
+
+There are three start-up states when using or launching the Cloud IDE:
+    
+- Creation start - This is the state in which you start the IDE for the first time. You can also view this as a *cold start* (see below), and you can expect this state to take longer because the git repository is being cloned.
```

Comment:
```suggestion
- Creation start - This is the state where you are starting the IDE for the first time. You can also view this as a *cold start* (see below), and you can expect this state to take longer because the git repository is being cloned.
```